components/form-token-field/index.js

```diff
@@ -503,6 +522,13 @@ FormTokenField.defaultProps = {
 	isBorderless: false,
 	disabled: false,
 	tokenizeOnSpace: false,
+	messages: {
+		added: __( 'Item added.' ),
+		removed: __( 'Item removed.' ),
+		remove: __( 'Remove item: %s.' ),
+	},
 };
 
-export default FormTokenField;
+FormTokenField.instances = 0;
```

Comment:
Why do we need to assign this property? Shouldn't this be handled by `withInstanceId`?